package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "version": "0.0.0",
   "description": "Netlify build module",
-  "main": "index.js",
```

Comment:
The top-level `package.json` is not published to `npm` and does not have any `main`/`exports` file.